backend/src/composite/mod.rs

```diff
@@ -157,9 +181,25 @@ impl<'a, F: FieldElement> Backend<'a, F> for CompositeBackend<'a, F> {
                     log::info!("== Proving machine: {} (size {})", machine, pil.degree());
                     log::debug!("PIL:\n{}", pil);
 
+                    let start = std::time::Instant::now();
```

Comment:
maybe move this to just before the call to `backend.prove`?

Reply:
Hmm, was thinking about it, but if the selecting the columns would be significant (hopefully not!) we'd want to measure it here, no? Note that also the inner backend typically logs the time it takes to prove, which might be slightly different if there is other stuff happening around it.